src/components/LoadingScreen/LoadingScreen.js

```diff
@@ -1,22 +1,76 @@
 // @flow
-import React, { PureComponent } from 'react';
+import React, { Fragment, PureComponent } from 'react';
 import { connect } from 'react-redux';
 import styled from 'styled-components';
+import IconBase from 'react-icons-kit';
+import { info } from 'react-icons-kit/feather/info';
 
-import { getBlockingStatus } from '../../reducers/app-status.reducer';
+import Spacer from '../Spacer';
+import ProgressBar from '../ProgressBar';
+import {
+  getBlockingStatus,
+  getStatusText,
+} from '../../reducers/app-status.reducer';
 
 import guppyLoaderSrc from '../../assets/images/guppy-loader.gif';
 import { COLORS, Z_INDICES } from '../../constants';
 
+const INFOTEXT_HEIGHT = 80;
+
 type Props = {
   showLoadingScreen: boolean,
+  statusText: string,
+};
+
+type State = {
+  showStatus: boolean,
 };
 
-class LoadingScreen extends PureComponent<Props> {
+class LoadingScreen extends PureComponent<Props, State> {
+  state = {
+    showStatus: false,
+  };
+
+  progress: number = 0;
+
+  toggleStatus = () => {
+    this.setState(state => ({
+      showStatus: !state.showStatus,
+    }));
+  };
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.statusText !== this.props.statusText) {
+      // new status
+      const [_, total] = (/\d+\/\d+/.exec(nextProps.statusText) || [
```

Comment:
nit: _ is unused

Reply:
`_` is a placeholder because I need the second value. 
Maybe I'll change this to use the index of the regex result.

Reply:
How about this? https://stackoverflow.com/a/44560070

Reply:
Yes, I've tried this too. It's working but I don't like how it looks.
I think I've seen the `_` destructuring in the Guppy code base before but it's better to avoid it. 
If changed it so it should be OK now.